internal/pkg/service/stream/storage/level/local/diskwriter/network/router/balancer/balancer.go

```diff
@@ -1,4 +1,4 @@
-package router
+package balancer
 
 import (
```

Comment:
I created new `balancer` and `pipeline` (sub) package, to split responsibilities.